src/main.rs

```diff
@@ -1,32 +1,25 @@
-extern crate env_logger;
-#[macro_use] extern crate hyper;
-#[macro_use] extern crate log;
+extern crate log;
+extern crate pretty_env_logger;
+extern crate reqwest;
 extern crate serde;
 extern crate serde_json;
 extern crate trust_dns;
 
-use hyper::Client;
-use hyper::header::Connection;
-
-use log::{LogLevelFilter};
-use env_logger::LogBuilder;
+use log::info;
 
 use serde_json::value::*;
 
 use std::env;
 use std::io;
 use std::io::prelude::*;
 
-use trust_dns::client::Client as DnsClient;
+use trust_dns::client::{Client as _Client, SyncClient};
```

Comment:
Can we call it `DnsClient` and `DnsSyncClient`? Prefixing with the underscore makes me think of an unused variable or a plugin-style import in Go. It sticks out in a bad way to me.

Reply:
It is an unused variable. However, it still needs to be imported because it is a trait for `SyncClient`.

Here's the build error I get if I remove it:
```
error[E0599]: no method named `query` found for type `trust_dns::client::SyncClient<trust_dns::udp::UdpClientConnection>` in the current scope
  --> src/main.rs:68:27
   |
68 |     let response = client.query(&name, DNSClass::IN, RecordType::TXT).unwrap();
   |                           ^^^^^
   |
   = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope, perhaps add a `use` for it:
   |
8  | use crate::trust_dns::client::Client;
   |

error: aborting due to previous error

For more information about this error, try `rustc --explain E0599`.
error: Could not compile `cfdyndns`.

To learn more, run the command again with --verbose.
```

Reply:
It's not really a variable and it's not unused in this case. I'd rather have

```diff
-use trust_dns::client::{Client as _Client, SyncClient};
+use trust_dns::client::{Client, SyncClient};
```

This builds for me on top of your PR. Thanks for the changes, if you make this last one I'll merge, or I can merge and add another commit if you want to be done with changes on it. Thanks!